teams/teams-ps/teams/Get-TeamTargetingHierarchyStatus.md

```diff
@@ -0,0 +1,89 @@
+---
+external help file: Microsoft.Open.Teams.CommonLibrary.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/connect-microsoftteams
+schema: 2.0.0
+author: mbergamo
+ms.author: mbergamo
+ms.reviewer:
+---
+# Get-TeamTargetingHierarchyStatus
+
+## SYNOPSIS
+
+Get the status of a hierarchy upload (see [Set-TeamTargetingHierarchy](https://docs.microsoft.com/powershell/module/teams/get-teamtargetinghierarchy))
+
+## SYNTAX
+
+### Get (Default)
+```
+Get-TeamTargetingHierarchyStatus [-RequestId <String>]
+```
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-TeamTargetingHierarchy -FilePath d:\hier.csv
+
+Key       Value
```

Comment:
Output from a cmdlet should be encased in ```output:

\```powershell
\> Cmdlet
\```

\```output
Result
\```